packages/fx-core/src/plugins/resource/aad/aadAppClient.ts

```diff
@@ -123,12 +143,22 @@ export class AadAppClient {
         );
       }
     } catch (error) {
-      throw ResultFactory.UserError(
+      if (
```

Comment:
seems previously it is always user error?

Reply:
Yes, since in my test I only found two errors:
1. sharing the bot, which will cause the same app id uri of two aad apps;
2. storage endpoint can not be set as app id uri;
And now it will classify the errors with status code.